### PR TITLE
oor: update to 1.3.0

### PR DESCRIPTION
**Pull request by albert-lopez**
Signed-off-by: Albert Lopez <alopez@ac.upc.edu>

Maintainer: Albert Lopez <alopez@ac.upc.edu>
Compile tested: (AR71xx, Netgear WNDR3800,18.06.2)
Run tested: (AR71xx, Netgear WNDR3800,18.06.2, Full tests)

Description: Update to new release of the package with support to DDT RF8111

### CODE REVIEW

File: net/oor/Makefile [hunk 1]
```diff
@@ -16,13 +16,13 @@
...
-PKG_VERSION:=1.2.2
-PKG_RELEASE:=1
+PKG_VERSION:=1.3.0
+PKG_RELEASE:=2
 
 
 PKG_SOURCE_URL=https://github.com/OpenOverlayRouter/oor/releases/download/$(PKG_VERSION)/
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_MD5SUM:=6702348fa314c2691f28c7e71337bb84
+PKG_MD5SUM:=5683570bff97b3c230327c6cda662ce2
```
Comment (diizzyy) [inline net/oor/Makefile]: This should be PKG_HASH:=<SHA256>

File: net/oor/Makefile [hunk 2]
```diff
@@ -16,13 +16,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=oor
-PKG_VERSION:=1.2.2
-PKG_RELEASE:=1
+PKG_VERSION:=1.3.0
+PKG_RELEASE:=2
```
Comment (diizzyy) [inline net/oor/Makefile]: No need to change ```PKG_RELEASE```

**Comment by albert-lopez** [comment]
I have done the suggested changes.

**Comment by diizzyy** [comment]
Please squash commits, looks good otherwise

**Comment by albert-lopez** [comment]
Done

**Pull request merged**